server/src/test/java/org/elasticsearch/search/aggregations/bucket/significant/SignificantTextAggregatorTests.java

```diff
@@ -245,7 +245,6 @@ public void testSignificanceOnTextArrays() throws IOException {
         }
     }
 
-    @AwaitsFix(bugUrl = "https://github.com/elastic/elasticsearch/issues/55360")
     @Override
```

Comment:
I think these overrides only exist so they can be muted, maybe we should delete them entirely?

Reply:
:+1: